web/apps/docs/src/content/docs/mim/pages/import.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Import Page
+---
+
+**_This page is only available to MIM Administrators_**
+
+MIM provides the ability to import data into the tool. There is a built-in import available that supports importing MIM ICDs that were exported from MIM.
+
+## MIM ICD Import
+
+The MIM ICD import option is available on all MIM installations. It is used to import ICDs that have been exported from MIM ([see reports for details](/org.eclipse.osee/mim/pages/reports#mim-icd)).
```

Comment:
Why are these hardlinks vs the relative links below?

Reply:
Routing requires hard paths, assets require relative paths